e2e/helpers/auth/backends/oidc.ts

```diff
@@ -14,8 +18,51 @@ type Secret = {
 // anything, it does not need to match the real controller names.
 const JIMM_CONTROLLER = "jimm-k8s";
 
+export class CreateOidcUser implements Action<OidcUser> {
```

Comment:
I wonder if the following suggestion sits well with the conventions we currently use. I leave it up to you to decide if we need to make this change :) 
```suggestion
export class CreateOIDCUser implements Action<OIDCUser> {
```